roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: try to fetch ceph config and keys
   copy:
-    src: "{{ playbook_dir }}/{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
```

Comment:
What's the reason for this change? We need to be careful because now we'll rely on the relative path `fetch/docker_mon_files/` instead of the absolute path provided by `playbook_dir`. Will this impact running playbooks from within `infrastructure-playbooks`?